query-service-impl/src/main/java/org/hypertrace/core/query/service/QueryRequestUtil.java

```diff
@@ -68,4 +71,23 @@ public static Expression createNullNumberLiteralExpression() {
                 .setValue(Value.newBuilder().setValueType(ValueType.NULL_NUMBER)))
         .build();
   }
+
+  public static Filter createContainsKeyFilter(String column, String value) {
+    return Filter.newBuilder()
+        .setOperator(Operator.CONTAINS_KEY)
+        .setLhs(createColumnExpression(column))
```

Comment:
Our new code should use attribute expressions since we're trying to deprecate column expressions.

Reply:
Support for `CONTAINS_KEY` and `CONTAINS_KEYVALUE` for attributeExpression will be added as a follow up PR.

Reply:
CONTAINS_KEYVALUE support we never need, CONTAINS_KEY should be trivial (I thought it was basically that one suggestion I made in another thread - will double check). If you want to do it in a separate PR that's fine, but we can't use attribute expressions until that support is added since the upstream won't be mixing column identifiers and attribute expressions - they'll all cut over together.

Reply:
Added this as a follow up PR task.